### PR TITLE
logging: fix up formatting initialization and remove duplicate timestamps

### PR DESCRIPTION
**Pull request by ianvernon**
With the addition of the default logger, formatting of logs was broken. Refactor
setupFormatter to return a logrus Formatter and use this formatter for the
default logger. Move logging initialization to the top of the cilium-agent's
initialization. Also stops logrus from adding its timestamp field to logs, as we
desire to have as clean and brief logs as possible; timestamping the logs is
left up to the logging framework to which Cilium logs (e.g., journalctl).

Signed-off by: Ian Vernon ian@cilium.io

Fixes: #1023

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]
logging bits LGTM.

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
do not merge until @raybejjani reviews this as well

File: pkg/endpoint/log.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func (e *Endpoint) updateLogger() {
 	// If this endpoint is set to debug ensure it will print debug by giving it
 	// an independent logger
 	if e.Opts.IsEnabled("Debug") {
-		baseLogger = logrus.New()
+		baseLogger = logging.InitializeDefaultLogger()
```
Comment (raybejjani) [inline pkg/endpoint/log.go:62]: Thank you! sorry for the not obvious random code.

**Review by raybejjani — APPROVED** [review]

**Pull request merged**